microshift_getting-started/microshift-understanding.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="con-microshift-understanding"]
+= Understanding MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-understanding
+
+toc::[]
+
+Get an overview of using {product-title}, a distribution of {OCP} designed for optimizing small form factors and edge computing.
```

Comment:
```suggestion
Get an overview of {product-title}, a distribution of {OCP} designed for optimizing small form factors and edge computing.
```
Minimalism suggestion

Reply:
The strategy for this doc is to focus more on the use/value in this section, the 'why.' I will update to be more clear that we are talking about what the customer can do with with MicroShift. The 'what' is in Architecture, the 'how' will be supporting sections.